docs/app3.rst

```diff
@@ -294,11 +294,81 @@ This design approach ensures that:
 
 .. _RedirectMessages:
 
-Redirecting Error and Warning Messages
---------------------------------------------
-Since MuPDF version 1.16 error and warning messages can be redirected via an official plugin.
+Diagnostics
+----------------------------
 
-PyMuPDF will put error messages to `sys.stderr` prefixed with the string "mupdf:". Warnings are internally stored and can be accessed via *pymupdf.TOOLS.mupdf_warnings()*. There also is a function to empty this store.
+.. _Messages:
+
+|PyMuPDF| messages.
```

Comment:
pedantic - we don't usually put full stops at the end of headers

Reply:
Thanks, fixed in my tree.